client.go

```diff
@@ -574,7 +575,7 @@ func (client *HTTPClient) do(req *Request) (err error) {
 
 	req.RawResponse, err = client.wrapped.Do(req.RawRequest)
 
-	client.logCall(req.RawRequest, req.RawResponse, start, err)
+	client.logCall(req.RawRequest, req.RawResponse, start, logCallErrorOrNil(err))
```

Comment:
I don ´t think that belongs here. This should go to the implementation of the logCall function. See: https://github.com/snabble/go-logging/blob/8fe41cb0acc101f69ed3ff10f991f7bf38a230af/helper.go#L163 for example.

Reply:
Also i think a context canceledhere means we have a timeout which is somthing the logger should know about for sure

Reply:
Makes sense. Thanks :)